### PR TITLE
readme: create a top-level readme to serve github

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+The Skydive Flow Exporter provides a framework for building pipelines which
+extract flows from the Skydive Analyzer (via it WebSocket API), process and
+send the results upstream.
+
+# Architecture
+
+The pipeline is constructed of a sequance of steps:
+- classify - tag flows according to some classification (such as ingress, egress)
+- filter - filter out flows, possibly based on classification
+- transform - transform Skydive Flow to any traget record type
```
Comment (dubek) [inline README.md]: s/traget/target/

File: README.md [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
-The IBM Security Advisor pipeline extracts flow data from skydive, filters out
-external traffic, transforms flow records by adding status information, and
-flow classification (egress, ingress, internal), batches flows records together
-into streams (by time or count), then encodes these blocks in JSON form, and
-compresses using GZIP and finally stores as objects into any S3 compatible
-target.
+# Overview
+
+The Skydive Flow Exporter provides a framework for building pipelines which
+extract flows from the Skydive Analyzer (via it WebSocket API), process and
```
Comment (dubek) [inline README.md]: s/process/process them/

File: README.md [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
+send the results upstream.
+
+# Architecture
+
+The pipeline is constructed of a sequance of steps:
+- classify - tag flows according to some classification (such as ingress, egress)
+- filter - filter out flows, possibly based on classification
+- transform - transform Skydive Flow to any traget record type
+- mangle - process a group of transformed flows, enrich data, add new resulting
+  flows, of delete redundent flows.
```
Comment (dubek) [inline README.md]: s/of/or/

**Comment by dubek** [comment]
Lines 72-76 will need to be updated, but we can do this after we merge this.

File: README.md (hunk 1 above)
Comment (hunchback) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (hunchback) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (hunchback) [reply README.md]: ok

**Comment by dubek** [comment]
Looks good for now as a first step, we'll need some more effort to make this more readable to people not really familiar with the project.

File: README.md [hunk 4]
```diff
@@ -1,11 +1,32 @@
...
-target.
+# Overview
+
+The Skydive Flow Exporter provides a framework for building pipelines which
+extract flows from the Skydive Analyzer (via it WebSocket API), process them
+and send the results upstream.
+
+# Architecture
+
+The pipeline is constructed of a sequance of steps:
```
Comment (lebauce) [inline README.md]: s/sequance/sequence/

File: README.md [hunk 5]
```diff
@@ -1,11 +1,32 @@
...
+- storeheader - add a header marker to a group of flows to be stored
+- store - handle grouping of flows to store object
+- encode - encode store object (JSON, CSV, etc.)
+- compress - compress encoded store object (GZIP, etc.) 
+- account - record accounting information of store object
+- write - write to target (S3, stdout, etc.)
+
+# Deploy
+
+We demonstrated deployment using IBM Security Advisor pipeline which  extracts
```
Comment (lebauce) [inline README.md]: nit: remove extra space

File: README.md [hunk 6]
```diff
@@ -1,11 +1,32 @@
...
+flow data from skydive, filters out external traffic, transforms flow records
```
Comment (lebauce) [inline README.md]: s/skydive/Skydive/

File: README.md (hunk 4 above)
Comment (hunchback) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (hunchback) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (hunchback) [reply README.md]: done

**Comment by hunchback** [comment]
ready for final review ...

**Comment by hunchback** [comment]
@safchain / @lebauce - please review / merge

**Pull request merged**